lib/Command/MigrateCustomGroups.php

```diff
@@ -167,11 +179,16 @@ public function updateShares(string $groupUri, string $circleId, array $memberId
 	 * manage local cache FederatedUser
 	 *
 	 * @param string $userId
-	 * @return FederatedUser
+	 * @return null|FederatedUser
 	 */
-	private function cachedFed(string $userId): FederatedUser {
+	private function cachedFed(string $userId): ?FederatedUser {
 		if (!array_key_exists($userId, $this->fedList)) {
-			$this->fedList[$userId] = $this->circlesManager->getLocalFederatedUser($userId);
+			try {
+				$this->fedList[$userId] = $this->circlesManager->getLocalFederatedUser($userId);
+			} catch (\Exception) {
```

Comment:
Maybe a bit broad to catch all exceptions here?
At least the exception should be logged, if you make it this broad.

Reply:
adding the exception to the warning